lib/controllers/game_controller.cpp

```diff
@@ -175,20 +180,42 @@ int GameController::getScreenHeight() const {
     return SCREEN_HEIGHT;
 }
 
-void GameController::startGame() {
+void GameController::startGame(int level_amount) {
```

Comment:
Level_amount is not used at this time, will it be used in the future or has it become obsolete during development?